static/skin/i18n/en.json

```diff
@@ -28,7 +28,7 @@
 	, "random-page-button-text": "Go to a randomly selected page"
 	, "searchbox-tooltip": "Search '{{BOOK_TITLE}}'"
 	, "confusion-of-tongues": "Two or more books in different languages would participate in search, which may lead to confusing results."
-	, "welcome-page-overzealous-filter": "No result. Would you like to <a href=\"?lang=\">reset filter</a>?"
+	, "welcome-page-overzealous-filter": "No result. Would you like to <a href=\"#lang=\">reset filter</a>?"
```

Comment:
The difference between the search (query) and fragment components in a URL is that when the search/query component changes the browser issues a new request and reloads the page, whereas for a changed fragment component the page isn't (re)loaded (the default action is to scroll the page to the anchor which the fragment component is supposed to stand for). You have to set-up a `window.onhashchange` event handler.

Reply:
Done.